lib/jxl/dec_external_image.cc

```diff
@@ -303,20 +314,27 @@ Status ConvertToExternal(const jxl::ImageBundle& ib, size_t bits_per_sample,
     color = &unpremul;
```

Comment:
Do we need to do this unpremul processing in the want_extra_channel case? I think this ends up not being used so we don't need to spend time here. This shows the problems of reusing this function to do something different when a bool parameter is true.

Reply:
No longer an issue after https://github.com/libjxl/libjxl/pull/338